src/roller.ts

```diff
@@ -58,9 +58,19 @@ export abstract class Roller<D, F, P> implements IRoller {
     }
 
     public rollCommand(command: string): string {
+        const parts = command.split('#');
+        let comment = '';
+        if (parts.length > 1) {
+          command = '' + parts.shift();
+          comment = '' + parts.join('#');
+        }
         const formula = command
             .replace(new RegExp(`/${this.command} `, 'g'), '');
-        return this.rollFormula(formula);
+        let result = this.rollFormula(formula);
+        if (comment !== '') {
+          result = `<div>${comment}</div>\n` + result;
```

Comment:
XSS security issue, this should also be in a template and escaped properly.

Reply:
Agreed. 
Now that I become aware of the optional flavourText, comment can simply be passed as flavourText, so escaping should no longer be an issue.

Reply:
this is the only thing left for me to test and merge it fyi :)

Reply:
thx. Also see update https://github.com/BernhardPosselt/foundryvtt-special-dice-roller/pull/8/commits/438206da95cb56d02453749f5ff5792689977b8b for this.